cmake/modules/AddSwift.cmake

```diff
@@ -799,9 +799,9 @@ function(add_swift_host_library name)
   add_dependencies(dev ${name})
   if(NOT LLVM_INSTALL_TOOLCHAIN_ONLY)
     swift_install_in_component(TARGETS ${name}
-      ARCHIVE DESTINATION lib${LLVM_LIBDIR_SUFFIX} COMPONENT dev
-      LIBRARY DESTINATION lib${LLVM_LIBDIR_SUFFIX} COMPONENT dev
-      RUNTIME DESTINATION bin COMPONENT dev)
+      ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}${LLVM_LIBDIR_SUFFIX} COMPONENT dev
+      LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}${LLVM_LIBDIR_SUFFIX} COMPONENT dev
```

Comment:
I don't think that this works.  We don't know if the path is something that we can tack on a 32/64 on to.  We should look at how clang deals with this.  Similar throughout.

Reply:
Why couldn't we append 32/64 to an arbitrary libdir? Clang doesn't deal with this, they do `lib${LLVM_LIBDIR_SUFFIX}`. 

https://github.com/llvm/llvm-project/blob/070283825aaff778f7fb527ebc4ef74c84b6c889/clang/cmake/modules/AddClang.cmake#L122

I could also scope the PR down to only the BINDIR since that's what we need for unblocking the Windows installer.

Reply:
Let's scope to what is needed in particular, and then address the libdir issue separately.

Reply:
Sounds good. I reverted the `LIBDIR` and `LIBEXECDIR` changes.